lib/core.js

```diff
@@ -35,6 +35,7 @@ declare class Object {
     static create(o: any, properties?: any): any; // compiler magic
     static defineProperties(o: any, properties: any): any;
     static defineProperty(o: any, p: any, attributes: any): any;
+    static entries<T>(object: {[key: string]: T}): Array<[string, T]>,
```

Comment:
@zertosh has two examples where this is broken: https://flowtype.org/try/#0PQKgBAAgZgNg9gdzCYAoVxhgEICUDyA0gKIByqAJgKYDGMAhgE5Vh30DO7Y+ARgFa0ALmADeqMBPaD6ggJY0wVAHaDGsquwA8AQQA0OAHwAKOPyEAuUQG0A1lQCel7QF1L2AL4BKJ40b17mlZ6OM4GANyo7qgAbkxg9PQ+fgFWAOQ0qfqpFKmhYAC83GY0ggB0yqrq7EY1SgCuMDCW9Er23kUCJZ6eERhYAOL4+AAi6NRszKwMnB1CouKS0nIKFWoamtjGpp2CliK2DpZSakoA5q44Xkn+gceyZ-rYoRFRsYxgPDzXKemZYNm5AwFWYlcoqNbVWoNJrxVrtXg7boRIA